WordPress/build.gradle

```diff
@@ -13,6 +13,57 @@ plugins {
     id "com.google.dagger.hilt.android"
     id "org.jetbrains.kotlinx.kover"
     id "com.google.devtools.ksp"
+    id "com.osacky.fladle"
+}
+
+fladle {
+    testTargets = [
+            "notPackage org.wordpress.android.ui.screenshots"
+    ]
+    configs {
+        wordpress {
+            def app = "wordpress"
+            debugApk.set(pathForVariant(app, "debugApk"))
+            instrumentationApk.set(pathForVariant(app, "instrumentationApk"))
+            smartFlankGcsPath.set(pathForSmartFlankGcsPath(app))
+            testTargets.set(testTargets.get() + [
+                    "notClass org.wordpress.android.e2e.StatsTests",
+                    "notClass org.wordpress.android.e2e.StatsGranularTabsTest",
```

Comment:
for test targets unavailable on the variant, do we need to define them here to be excluded? 

asking because when looking at the tests, it looks like there's a check to see if it's jetpack app before continuing with test setup: https://github.com/wordpress-mobile/WordPress-Android/blob/a93b99968d4497ade86305d6e144457dc1170504/WordPress/src/androidTest/java/org/wordpress/android/e2e/StatsGranularTabsTest.kt#L24

Reply:
Unfortunately, they have to. If they're not, `fladle/flank` might create a shard with only tests from `StatsTests` or `StatsGranularTabsTest`. As they're ignored, the execution will be very fast (few seconds) and Firebase Test Lab will mark the test run as failed. It will do this to signal a problem, as too aggressive sharding might bring more unwanted costs. Some more details are available in here the comment description: cc4cc3b . Failed run like this can be found here https://buildkite.com/automattic/wordpress-android/builds/19467#01910e2f-5783-4f74-8897-2725f66ce343

<img width="525" alt="image" src="https://github.com/user-attachments/assets/94b3f357-38be-47ef-8d7e-81b478d48d3d">

Reply:
I see, thanks for the detailed explanation and linking that commit's comment description! What do you think about adding a comment on the test too? So we would know for a future test that's only available on one variant we should add it to the exclusion list too

Reply:
Sure thing, added a comment! WDYT? 9c05ddd98f2b814921613ea56dbf3d57cc60ca2f